jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java

```diff
@@ -49,14 +49,15 @@
 import org.springframework.test.context.DynamicPropertyRegistry;
 import org.springframework.test.context.DynamicPropertySource;
 import org.springframework.test.context.TestPropertySource;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
 import org.testcontainers.containers.MSSQLServerContainer;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
```

Comment:
these assertions can be replaced also with the junit5 assertions counterparts instead.

`org.junit.jupiter.api.Assertions`

Reply:
good catch @afalhambra ! thanks a lot, updated